oak_restricted_kernel/src/lib.rs

```diff
@@ -56,6 +57,7 @@ pub mod shutdown;
 #[cfg(feature = "simple_io_channel")]
 mod simpleio;
 mod snp;
+mod syscall;
```

Comment:
What is this library doing exactly?
Is it actually trying to execute syscalls? Or it's just a list of wrappers?

Reply:
This bit inside Restricted Kernel itself is the syscall implementations themselves.

Whenever user code calls the `SYSCALL` instruction (see `oak_restricted_kernel_runtime/src/raw_syscall.rs`) the CPU jumps to `syscall_entrypoint` in `oak_restricted_kernel/src/syscall/mod.rs`.